doc/reference/scipy_special.rst

```diff
@@ -15,6 +15,5 @@ Error function and Fresnel integrals
    erf
    erfc
    erfcx
-   erfi
```

Comment:
What is the reason for deleting this line?

Reply:
No plan to support that as for now. CuPy also doesn't have it.